examples/app_simple_tdm_master/src/simple_tdm_master.xc

```diff
@@ -18,8 +16,9 @@ void my_application(server tdm_callback_if i_tdm) {
   while (1) {
     select {
       case i_tdm.init(i2s_config_t &?i2s_config, tdm_config_t &?tdm_config):
-        i2s_config.mclk_bclk_ratio = (MASTER_CLOCK_FREQUENCY/SAMPLE_FREQUENCY)/64;
-        i2s_config.mode = I2S_MODE_LEFT_JUSTIFIED;
+        tdm_config.offset = 0;
```

Comment:
How did this ever work before?!

Reply:
good question...

Reply:
actually its still wrong..

Reply:
no its correct now I think. i don't think it would ever work before but trap at run time since i2c_config is null